src/Domain/Configuration.php

```diff
@@ -233,6 +231,11 @@ public function getFileLinkFormatter(): FileLinkFormatterContract
         return $this->fileLinkFormatter;
     }
 
+    public function isFix(): bool
```

Comment:
Consider if `shouldFix` or `hasFix` is a better name or something else. `isFix` sounds a little weird. Or something completely different.

Reply:
I reword it `hasFixEnabled`. Good for you ?

Reply:
Perfect :)